### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by igor-kashchenko**
- [DEMO LINK](https://igor-kashchenko.github.io/layout_catalog_hovers/)

### CODE REVIEW

**Review by Kardash-Oleksandr — APPROVED** [review]
Good job)

**Comment by igor-kashchenko** [comment]
> Good job)

Thank you for your comment!

File: src/styles/blocks/catalog.scss [hunk 1]
```diff
@@ -18,6 +18,13 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: transform $transition-duration, color $transition-duration;
+
+    &:hover {
+      transform: scale(1.2);
+      color: #34568b;
```
Comment (alexandra-protyanova) [inline src/styles/blocks/catalog.scss:25]: you can move color to variable as well

**Review by alexandra-protyanova — APPROVED** [review]
Well done!

**Pull request closed without merging**